cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -195,6 +196,14 @@ func newApplyData(cmd *cobra.Command, args []string, applyFlags *applyFlags) (*a
 		return nil, cmdutil.TypeMismatchErr("dryRun", "bool")
 	}
 
+	// If dry running creates a temporary directory for saving kubeadm generated files.
+	dryRunDir := ""
+	if *dryRun {
+		if dryRunDir, err = constants.GetDryRunDir(constants.EnvVarUpgradeDryRunDir, "kubeadm-upgrade-apply-dryrun", klog.Warningf); err != nil {
+			return nil, errors.Wrap(err, "could not create a temporary directory")
```

Comment:
```suggestion
			return nil, errors.Wrap(err, "couldn't create a temporary directory on dryrun")
```
Will this be better? Just want to avoid duplicating error messages with the code [here](https://github.com/kubernetes/kubernetes/blob/aa819af85a1607a3642a6fb4bbb4ff978079e692/cmd/kubeadm/app/constants/constants.go#L639).

Reply:
a clarification seems fine, but we should use the long form "could not", "would not" everywhere.